app/assets/javascripts/discourse/app/templates/components/emoji-group-buttons.hbs

```diff
@@ -1,30 +1,30 @@
 {{!-- DO NOT EDIT THIS FILE!!! --}}
 {{!-- Update it by running `rake javascript:update_constants` --}}
```

Comment:
There's a template in lib/tasks/javascript.rake that needs to be updated 😃

Reply:
This file was [updated here](https://github.com/discourse/discourse/pull/18333/files#diff-3131ea53afd0c860ac0ccf3fed0975a8633945dc2a3e1e4f6960ec24af465446). And when I run `rake javascript:update_constants` there are no diffs.

Are you seeing something out of sync?

Reply:
I just updated the template in this rakefile to use `fn` instead of `action` and re-ran the task.